openstack/networking/v2/extensions/attributestag/results.go

```diff
@@ -0,0 +1,26 @@
+package attributestag
+
+import (
+	"github.com/gophercloud/gophercloud"
+)
+
+// "tagResult" denoting that "tags" are returned and can be used for both the ReplaceAll and Get actions
+
+type tagResult struct {
+	gophercloud.Result
+}
+
+// Extract interprets any tagResult as Tags.
```

Comment:
```
// Extract interprets any tagResult as a slice of strings
```

The use of `Tags` in this method is only used internally and isn't exposed to the user. The user only sees the slice.